pages/community/ambassadors/index.js

```diff
@@ -249,7 +234,7 @@ function Index() {
               </Heading>
               <Heading typeStyle="body-sm">
                 The AsyncAPI Ambassador program is now open for applications! If
-                you’re selected, you’ll join AsyncAPI's mission of helping
+                you&rsquo;re selected, you&rsquo;ll join AsyncAPI&apos;s mission of helping
```

Comment:
@yuvrxj-afk Why did you make this change? Does it change anything on the UI?

Reply:
It doesn't, getting the reds under it so , added the html entities. 
Can remove them as well @akshatnema.
The UI is still same.

Reply:
It doesn't, getting the reds under it so , added the html entities. 
Can remove them as well @akshatnema &
The UI is still same.

Reply:
Please remove these changes from the PR as it will look quite awkward while reading the plain markdown file from the code point of view.